services/scan/src/store.ts

```diff
@@ -946,6 +949,8 @@ export class Store {
         writeStream.write('\n');
       }
     }
+
+    writeStream.end();
```

Comment:
didn't end up being important after I landed on using the `Passthrough` stream, but still makes sense to close the stream